ide/che-core-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/OrionEditorPresenter.java

```diff
@@ -1149,6 +1172,58 @@ public void onContextMenu(ContextMenuEvent event) {
     }
   }
 
+  private Promise<Void> initializeChangeMarkersRender() {
+    OrionTextViewOverlay textView = editorWidget.getTextView();
+    List<OrionExtRulerOverlay> rulers = Arrays.asList(textView.getRulers());
+
+    OrionStyleOverlay style = OrionStyleOverlay.create();
+    style.setStyleClass("vcsChangeMarker");
```

Comment:
If you set `vcsChangeMarker` style why you don't define everything in css instead of setting attributes?

Reply:
I would like to have style class name similar to another rules: "ruler vcs"

Reply:
> If you set vcsChangeMarker style why you don't define everything in css instead of setting attributes?

We have `orion-codenvy-theme.css` that holds styles for default rulers, but style from that css does not affect to new ruler. `OrionStyleOverlay` does not support injecting css files.

> I would like to have style class name similar to another rules: "ruler vcs"

done

Reply:
> If you set vcsChangeMarker style why you don't define everything in css instead of setting attributes?

Fixed, it started to work after changing the ruller style class name to `ruler vcs`